packages/block-editor/src/components/global-styles/typography-panel.js

```diff
@@ -52,52 +39,38 @@ export function useHasTypographyPanel( name, element, settings ) {
 	);
 }
 
-function useHasFontSizeControl( name, element, settings ) {
-	const supports = useSupportedStyles( name, element );
```

Comment:
Now this function is only called once for the "TypographyPanel" rendering, there might be a small perf improvement here (very hard to measure). I believe this addresses one of the points raised by @jorgefilipecosta in the original PR.